Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM nginx:1.17.5
 
-ENV GIT_URL https://github.com/kadena-community/kadena-transfer-js
+ENV GIT_URL https://github.com/obsidiansystems/kadena-transfer-js.git
```

Comment:
Can we change this back

Reply:
https://github.com/obsidiansystems/kadena-transfer-js/pull/5